.gitlab-ci.yml

```diff
@@ -0,0 +1,32 @@
+stages:
+  - build
+  - deploy
+
+build:
+  tags: ["runner:docker", "size:large"]
+  stage: build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker:v3892323-786b6fd-20.10.3
```

Comment:
Where did you get this image? It's not found in the canonical repository in the canonical place. It doesn't have the ecr credential helpers push images, as I recently found out in libddprof.